### ISSUE TITLE
Missing h1 element in home info mode

### ISSUE THREAD

**Issue by fkohrt** [adityatelange/hugo-PaperMod#199]
I did some accessibility testing and [WAVE complains](https://wave.webaim.org/report#/https://adityatelange.github.io/hugo-PaperMod/) about a _Missing first level heading_. So maybe change the following `h2` to an `h1` element?

https://github.com/adityatelange/hugo-PaperMod/blob/353146891e2e3c38600b406efd904d35b5debc86/layouts/partials/home_info.html#L3-L5
